website/blog/2021-08-17-interview-airwallex.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Apache APISIX 在 Airwallex 的应用 | 专访 Airwallex 技术平台负责人李杨"
+keywords: 
+- APISIX
+- Airwallex
+- 空中云汇
+- 金融
+- 数据主权
+description: 本文采访了 Airwallex 空中云汇 Technical Platform Lead 李杨，李杨负责公司技术平台的演进。采访中详细介绍了 Airwallex 空中云汇在做技术选型时为什么选择 Apache APISIX、Apache APISIX 在 Airwallex 空中云汇的使用场景、Apache APISIX 在生产环境的表现等。
+tags: [technology]
+---
+
+<!--truncate-->
+
+我们有幸采访到了 Airwallex 的技术平台负责人李杨，在采访中，李杨谈到了为什么在技术选型时选择 Apache APISIX ，以及 Apache APISIX 在 Airwallex 的应用。
+
+**Q：李博士，您好，请简单介绍一下自己和目前从事的工作。**
+
+**李杨：** 你好，我叫李杨（Jan Li），哲学博士，Apache APISIX Committer, Airwallex 空中云汇 Technical Platform Lead，负责公司技术平台的演进。在加入 Airwallex 之前，曾在万向区块链领导运链盟。在万向区块链之前，曾在花旗集团领导 OTC 衍生品风控平台。
```

Comment:
Apache APISIX Committer,  ---> Apache APISIX Committer，

comma should change to Chinese

Reply:
you might want to double check all symbols as well :;'",.()

Reply:
thanks